app/templates/bpmn-files/bpmn-file/index.hbs

```diff
@@ -60,6 +60,13 @@
                   {{this.metadata.publisher.name}}
                 </:content>
               </Item>
+
+              <Item>
+                <:label>Contact Email</:label>
```

Comment:
As you mentioned, "Contact Email" doesn't sound perfect in Dutch. However, it's not very easy to find a better translation. Maybe we should just call it "Contact"? It takes less space and it's probably clear to the user that these are email addresses?

Reply:
Idd! I've renamed it to contact, will deploy tomorrow on dev hopefully